mappings/net/minecraft/util/LowercaseEnumTypeAdapterFactory.mapping

```diff
@@ -0,0 +1,3 @@
+CLASS net/minecraft/class_3530 net/minecraft/util/LowercaseEnumTypeAdapterFactory
+	METHOD method_15334 getKey (Ljava/lang/Object;)Ljava/lang/String;
```

Comment:
Why isn't this json?

Reply:
Because the json package is gone.

Reply:
The classes were re-arranged to address @Juuxel 's comment and that only left two classes that were strictly "json stuff" which rendered the purpose of having a /uril/json package at all a moot point.

Reply:
The classes were re-arranged to address @Juuxel 's comment and that only left two classes that were strictly "json stuff" which rendered the purpose of having a /uril/json package at all a moot point.

Reply:
The classes were re-arranged to address @Juuxel 's comment and that only left two classes that were strictly "json stuff" which rendered the purpose of having a /uril/json package at all a moot point.